presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/SchemaDao.java

```diff
@@ -130,4 +131,11 @@
             "  FOREIGN KEY (node_id) REFERENCES nodes (node_id)\n" +
             ")")
     void createTableBuckets();
+
+    @SqlUpdate("CREATE TABLE IF NOT EXISTS shard_organizer (\n" +
+            "  node_identifier VARCHAR(255) NOT NULL,\n" +
+            "  last_start_time DATETIME NOT NULL,\n" +
```

Comment:
Based on how this is used, it should be named `last_run_time`, since it is actually updated at the end of the run, not the start